app/src/scripts/convert.js

```diff
@@ -82,6 +82,19 @@ function convertToWebm(opts) {
   });
 }
 
+// Should be similiar to the Gif generation
+function convertToApng(opts) {
+  return Promise.resolve().then(() => {
+    return convert(opts.outputPath, opts, [
+      '-i', opts.filePath,
```

Comment:
You need to set the framerate and width/height too, like done with the GIF.

https://www.ffmpeg.org/ffmpeg-formats.html#apng

Or maybe try using:

```
'-filter_complex', `fps=${opts.fps},scale=${opts.width}:${opts.height}:flags=lanczos[x];`
```